### ISSUE TITLE
Feature Request: Improve Lighthouse Best Practice Score

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#101]
### Describe the feature

The site currently has a score of 86 for Best Practices when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110366597-6d005c80-803e-11eb-8e57-c8dff72ac56e.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the practices of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for best practices is increased to at least 90.
